apps/yggdrasil/priv/repo/migrations/20160229163000_create_role_resource.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Yggdrasil.Repo.Migrations.CreateRoleResource do
+  use Ecto.Migration
+
+  def change do
+    create table(:roles_resources) do
+      add :role_id, references(:roles, on_delete: :nothing)
+      add :resource_id, references(:resources, on_delete: :nothing)
+      add :permission_id, references(:permissions, on_delete: :nothing)
+
+      timestamps
+    end
+    create index(:roles_resources, [:role_id])
+    create index(:roles_resources, [:resource_id])
+    create index(:roles_resources, [:permission_id])
```

Comment:
@copenhas is there a need for the last two indexes here? I don't see very many needs to look up the role_resource record by resource_id or permissions_id - it will pretty much always be driven by role_id I think.